app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/optionsStepForm/optionsBlock/optionsBlock.jsx

```diff
@@ -0,0 +1,144 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { InputRadioGroup } from 'components/inputs/inputRadioGroup';
+import classNames from 'classnames/bind';
+import { TO_INVESTIGATE } from 'common/constants/defectTypes';
+import { useSelector } from 'react-redux';
+import { activeFilterSelector } from 'controllers/filter';
+import { defectTypesSelector } from 'controllers/project';
+import {
+  CURRENT_EXECUTION_ONLY,
+  LAST_TEN_LAUNCHES,
+  SEARCH_MODES,
+  SIMILAR_TI_CURRENT_LAUNCH,
+  WITH_FILTER,
+} from '../../../constants';
+import { messages } from '../../../messages';
+import styles from './optionsBlock.scss';
+
+const cx = classNames.bind(styles);
+
+export const OptionsBlock = ({
+  optionValue,
+  collapseTabsExceptCurr,
+  currentTestItem,
+  loading,
+  setModalState,
+}) => {
+  const { formatMessage } = useIntl();
+  const activeFilter = useSelector(activeFilterSelector);
+  const defectTypes = useSelector(defectTypesSelector);
+  const TIDefectsGroup = defectTypes[TO_INVESTIGATE.toUpperCase()];
+  const getOptions = () => {
+    const currentItemFromTIGroup = TIDefectsGroup.find(
+      (type) => type.locator === currentTestItem.issue.issueType,
+    );
+    const options = [
+      {
+        ownValue: CURRENT_EXECUTION_ONLY,
+        label: {
+          id: CURRENT_EXECUTION_ONLY,
+          defaultMessage: formatMessage(messages.currentExecutionOnly),
+        },
+      },
+    ];
+    if (currentItemFromTIGroup) {
+      const optionalOptions = [
+        {
+          ownValue: SIMILAR_TI_CURRENT_LAUNCH,
+          label: {
+            id: SIMILAR_TI_CURRENT_LAUNCH,
+            defaultMessage: formatMessage(messages.similarTICurrentLaunch),
+          },
+        },
+        {
+          ownValue: LAST_TEN_LAUNCHES,
+          label: {
+            id: LAST_TEN_LAUNCHES,
+            defaultMessage: formatMessage(messages.lastTenLaunches),
+          },
+        },
+      ];
+      activeFilter &&
+        optionalOptions.push({
+          ownValue: WITH_FILTER,
+          label: {
+            id: WITH_FILTER,
+            defaultMessage: formatMessage(messages.withFilter, {
+              filterName: activeFilter.name,
+            }),
+          },
+        });
+      options.push(...optionalOptions);
+    }
+    return options;
+  };
+  const onChangeOption = (value) => {
+    let searchMode;
+    switch (value) {
+      case SIMILAR_TI_CURRENT_LAUNCH:
```

Comment:
Use just `SEARCH_MODES.SIMILAR_TI_CURRENT_LAUNCH` and remove redundant constants